src/resampling.jl

```diff
@@ -30,7 +30,9 @@ abstract type EnsembleDistribution end
 function ensemble_normal_distribution(X)
     μ = [mean(X, dims=2)...]
     Σ = cov(X, dims=2)
-    return MvNormal(μ, Σ)
+    Σᴴ = Matrix(Hermitian(Σ))
+    @assert Σᴴ ≈ Σ 
```

Comment:
If this is not true, we get an error on the following line, correct?

`@assert` is really a debugging tool rather than a check for correctness (it gets disabled at some optimization levels). What are we trying to express here?

Reply:
Yeah that was for debugging. I removed it.